users/README.md

```diff
@@ -67,7 +67,7 @@ future changes by simply running `terraform apply
 | assume_any_role_anywhere_policy_description | The description to associate with the IAM policy that allows assumption of any role in any account, so long as it has a trust relationship with the Users account. | string | `Allow assumption of any role in any account, so long as it has a trust relationship with the Users account.` | no |
 | assume_any_role_anywhere_policy_name | The name to assign the IAM policy that allows assumption of any role in any account, so long as it has a trust relationship with the Users account. | string | `AssumeAnyRoleAnywhere` | no |
 | aws_region | The AWS region where the non-global resources for this account are to be provisioned (e.g. us-east-1) | string | `us-east-1` | no |
-| godlike_usernames | The usernames associated with the god-like accounts to be created, which are allowed to access the terraform backend, are IAM administrators for the Users account, and are allowed to assume any role that has a trust relationship with the Users account.  The format first.last is recommended.  The keys are the usernames and the values are empty strings (since they are not presently used). Example: { \"firstname1.lastname1\" = \"\",  \"firstname2.lastname2\" = \"\" } | map(string) | | yes |
+| godlike_usernames | The usernames associated with the god-like accounts to be created, which are allowed to access the terraform backend, are IAM administrators for the Users account, and are allowed to assume any role that has a trust relationship with the Users account.  The format first.last is recommended.  The keys are the usernames and the values are empty strings (since they are not presently used). Example: ["firstname1.lastname1", "firstname2.lastname2"] | list(string) | | yes |
```

Comment:
Remove "The keys are the usernames and the values are empty strings (since they are not presently used)."

Reply:
Good catch, thanks! Fixed in https://github.com/cisagov/cool-accounts/pull/29/commits/99b05954723f934a6c5f3e855240f23d34d20941.